pkg/params/info/events.go

```diff
@@ -43,6 +45,10 @@ type Event struct {
 	// Bitbucket Server
 	CloneURL string // bitbucket server has a different url for cloning the repo than normal public html url
 	Provider *Provider
+
+	// Gitlab
+	SourceProjectID int
```

Comment:
we worked so hard to remove provider specific info in Event and now they are back 🙃

Reply:
yeah... i was thinking to add a `AddYourStuff` kind of function for providers
who will add their stuff to pipelinerun